unitTests/ui/ResultLayoutSelectorTest.ts

```diff
@@ -35,6 +36,36 @@ export function ResultLayoutSelectorTest() {
       return test.cmp['currentActiveLayouts'][layout].button.el;
     }
 
+    it('should filter activelayouts based on the desktopLayouts options', () => {
```

Comment:
I think it would be useful to mention the screen size in the descriptions of the three tests:

```
when the screen is large, activeLayouts returns the desktop layouts
```